test/dlc_tests/benchmark/ec2/tensorflow/inference/test_performance_tensorflow_inference.py

```diff
@@ -35,16 +42,12 @@ def ec2_performance_tensorflow_inference(image_uri, processor, ec2_connection, r
     )
     time_str = time.strftime('%Y-%m-%d-%H-%M-%S')
     commit_info = os.getenv("CODEBUILD_RESOLVED_SOURCE_VERSION")
-    log_file = f"inference_benchmark_results_{commit_info}_{time_str}.log"
+    log_file = f"synthetic_{commit_info}_{time_str}.log"
```

Comment:
Will naming this log file the same for all images being tested in parallel cause the scripts to overwrite one another's logs in the same file (if time_str doesn't change between two parallel tests)?